### PR TITLE
Created converter for the conversion between poloidal and toroidal flux lable s_pol and s_tor

### PR DESCRIPTION
**Pull request by GeorgGrassler**
Created converter for the conversion between poloidal and toroidal flux lable s_pol and s_tor

### CODE REVIEW

**Comment by pazathoth** [comment]
The Python looks nice (though a space after comma lists wouldn't hurt), but can we please call it label and not lable?

**Comment by GeorgGrassler** [comment]
> The Python looks nice (though a space after comma lists wouldn't hurt), but can we please call it label and not lable?

@pazathoth Yeah, that spelling is on me. Is a common issue of mine with these le/el endings (._.) ... fixed it now, but what do you mean specifically with your "space after comma lists"? I do not see in the code which part you mean ...

File: python/libneo/flux_label_converter.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+        spol = self.interp_spol(stor)
+
+        return spol
+
+################################################################################
+################################################################################
+################################################################################
+
+if __name__=='__main__':
```
Comment (krystophny) [inline python/libneo/flux_label_converter.py:60]: Can you move this into an actual `pytest` test? See https://chat.openai.com/share/e49e67a1-6caf-4448-abca-474b10e34acc

**Review by krystophny — CHANGES_REQUESTED** [review]

File: python/libneo/flux_label_converter.py (hunk 1 above)
Comment (GeorgGrassler) [reply python/libneo/flux_label_converter.py:60]: Made the modification. Please check if okay now.

**Review by krystophny — APPROVED** [review]

**Review by pazathoth — APPROVED** [review]

**Pull request merged**